pages/getting-started/installation-guide/index.md

```
@@ -51,16 +51,17 @@ With your favorite browser, visit the URL `http://localhost:8000`
 ---
 
 <h2 id="vagrant">Vagrant</h2>
-Run Vagrant from the official [Vagrant Build](https://pilab.dev/bludit-vagrant).
+Run Vagrant from the official [Vagrant Build](https://jereme.dev/bludit-vagrant).
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)